source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,286 @@
+.. _swupd-3rd-party:
+
+Create, Install, and Manage 3rd-party Bundles
+#############################################
+
+Upstream Clear Linux offers a plethora of bundles to choose from.  
+As of version 32570, there are over 1000 bundles and counting.  
+
+To provide you access to even more software outside of the distro, we have 
+enabled two new features to our :command:`mixer` and :command:`swupd` client tools
+starting with release 32570.  
+
+:command:`mixer` can now create 3rd-party bundles out of any content you want 
+without the need to package it into an RPM format first.  
+
+And to consume and manage these bundles, we have added a `3rd-party` 
+subcommand to :command:`swupd`.    
+
+Follow this guide to see how to build an example 3rd-party bundle, set up a web 
+server to host it, install and manage the bundle on a client system. 
+
```

Comment:
It would be good to mention what sorts of things 3rd-party *cannot* do early on.

Reply:
Added a "some limitations" section at the bottom.